test/tests/userobjects/nearest_point_layered_average/tests

```diff
@@ -60,4 +60,21 @@
     design = 'NearestRadiusLayeredAverage.md'
     issues = '#16829'
   [../]
+
+  [./points_from_uo]
+    type = 'CSVDiff'
+    input = 'points_from_uo.i'
+    csvdiff = 'points_from_uo_out_spatial_from_uo_0002.csv'
+    requirement = 'The system shall compute layered averages computed from the closest values for a list of points and output to a spatial vector postprocessor with one point per unique nearest point and layer'
+    design = 'NearestPointLayeredAverage.md'
+    issues = '#18931'
+  [../]
+  [./radius_points_from_uo]
+    type = 'CSVDiff'
+    input = 'radius_points_from_uo.i'
+    csvdiff = 'radius_points_from_uo_out_spatial_from_uo_0002.csv'
+    requirement = 'The system shall compute radial layered averages computed from the closest values for a list of points and output to a spatial vector postprocessor with one point per unique nearest point and layer'
+    design = 'NearestRadiusLayeredAverage.md'
+    issues = '#18931'
+  [../]
```

Comment:
use `[test_name]` and `[]` instead. And if you could update the rest of the `tests` file as well, please -  `./` and `../` needs to be eradicated - this will help with that...